Gems/GradientSignal/Code/Source/Components/ImageGradientComponent.cpp

```diff
@@ -707,7 +707,7 @@ namespace GradientSignal
 
     void ImageGradientComponent::OnAssetMoved(AZ::Data::Asset<AZ::Data::AssetData> asset, [[maybe_unused]] void* oldDataPointer)
     {
-        OnAssetReady(asset);
+        //OnAssetReady(asset);
```

Comment:
Rather than comment this out, just remove the entire override. There's no value in leaving it here with commented-out code.